docs/configuration/configuration-file.mdx

```diff
@@ -5,224 +5,465 @@ import { StyledContent } from '../../components/styled.elements';
+Bonus: You only need to type `name` or `name@version` in commands, not the entire path to the file. No one's getting any younger — especially not us. Where we can save you time, we will!
 
-resolve:
-# Configuration for resolving external links in your definition that are not publicly accessible.
-# Not required for Redocly API registry links.
+### Summary of `apis` properties
 
-features.openapi:
-# The API reference docs options, including theme options.
+| Property |  | Type | Description |
+|---|---|---|---|
+| `{apiName}` |  | `patternProperties` | REQUIRED. Identifies an API definition file. |
+| `{apiName}@{apiVersion}` |  | `patternProperties` | Identifies multiple versions of an API definition file. `{apiName}` is required but `@version` is optional. |
+|  | `root` | `string` | REQUIRED. The relative path to where a definition file is stored. |
+|  | `labels` | `array` | Assigns Redocly Workflows organization-wide labels to a definition file. |
+|  | `lint` | `object` | Defines linting rules at the file level (overrides global `lint` configuration). |
+|  | `features.openapi` | `object` | Configures theming and functionality for an API definition. |
 
-registry:
-# The API registry options
 
-# future products may be configured by extending this file
+### Examples
+
+```yaml
+apis:
+  main:
+    root: ./openapi.yaml
+    labels:
+      - main
+    lint: []
+    features.openapi: []
+  production:
+    root: ./production.yaml
+    labels:
+      - production
+    lint: []
+    features.openapi: []
+  sandbox:
+    root: ./sandbox.yaml
+    labels:
+      - sandbox
+    lint: {}
+    features.openapi: {}
 ```
 
+Given the configuration above and the following command, `./production.yaml` is validated because it's the only one currently using the name `production`:
 
-### organization
+```shell
+openapi lint production
+```
 
-The `organization` property requires the organization ID of your Redocly Workflows organization.
-When specified in the configuration file, it can be used by the `openapi push` command.
-The `push` command uses the following order of precedence: first, it takes the organization ID from command-line arguments (if provided).
-If the organization ID is not provided explicitly, it takes it from the configuration file.
+If you run `lint` without specifying a name, all API definitions listed in the `apis` section will get validated:
 
-Note that the organization ID is not the same as the organization name in Workflows.
+```shell
+openapi lint
+```
 
-To find the organization ID, log into Workflows and access the **API registry** page.
-In your browser's address bar, find the URL of the page.
-The segment after `app.redocly.com/org/` is your organization ID.
-For example, if the URL is `app.redocly.com/org/test_docs`, the organization ID is `test_docs`.
+When you run the [preview-docs command](../commands/preview-docs.md), you *must* specify a file. As a shortcut, you can reference definitions listed in the `apis` section.
```

Comment:
question: why are we focusing specifically on the `push` command here? Isn't this true for most commands?

Reply:
Did you mean the `preview-docs` command? That was in the original so I kept it. It sounds to me like you *can* specify a file in any command, but you *must* specify a file in the `preview-docs` command.

@adamaltman - can you please confirm?

Reply:
My bad, you're right, it's `preview-docs`. Still, the same question stands... `preview-docs `doesn't have to be used with a file. It also works with that "shortcut" of referencing the `apiName@version` syntax. It's documented here: https://redocly.com/docs/cli/commands/preview-docs/#pass-entrypoint-alias

Reply:
I can't find this in the original text, could you please point me to where it is? That's one of the reasons why I'm so confused by this sentence :sweat_smile:

Reply:
No, you can use an API name@version too.

Reply:
@skadinna now I am the one who is confused! 😅 I don't know why I said that was in the original - it's not. Sorry for the wild goose chase!

I will change that entire statement about `preview-docs` to make it just another example of how definition file names can be passed in any command.